ProjectSidewalk/SidewalkWebpage
# Resolves #1511: Updates tree obstacle in path example

Resolves #1511.
* Updated validation obstacle in path tree example.

**Testing Instructions**
* Load an Obstacle in Path validation mission. Hover over the image to check that it looks okay. Not expecting a lot to change here, since I didn't update any code.

This is what the old tree example looked like:
![image](https://user-images.githubusercontent.com/25534091/55842276-96797c00-5ae7-11e9-834a-79276df898ea.png)


These are the four examples for obstacle in path.
![image](https://user-images.githubusercontent.com/25534091/55842238-73e76300-5ae7-11e9-9995-185b4bd5a009.png)

This is what it currently looks like when you hover over the new tree example.
![image](https://user-images.githubusercontent.com/25534091/55842220-603bfc80-5ae7-11e9-9431-b7bd59672397.png)
